starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express')
+
+const celebrityModel = require('../models/celebrity')
+
+const router = express.Router()
+
+
+router.get('/celebrities', (req,res, next)=>{
+    celebrityModel.find()
+    .then((celebrity) =>{
+        console.log(celebrity)
+        res.render('celebrities/index.hbs', {celebrity})
+    })
+    .catch((err) =>{
+        console.log("Problem with the collection", err)
+    })
+})
+
+
+router.get('/celebrities/new', (req,res, next)=>{
+    res.render('celebrities/new.hbs')
+
+ })
+
+router.post('/celebrities/new', (req, res) =>{
+ celebrityModel.create(req.body)
```

Comment:
note that you have different indentation levels as the routes above. Remember to check for code cleanness and indentation. It will help you greatly in your projects. Check the auto-format feature of VS code with prettier.